Cython/Compiler/Parsing.pxd

```diff
@@ -64,6 +64,8 @@ cdef expect_ellipsis(PyrexScanner s)
 cdef make_slice_nodes(pos, subscripts)
 cpdef make_slice_node(pos, start, stop = *, step = *)
 cdef p_atom(PyrexScanner s)
+cdef parse_atom_string(pos, PyrexScanner s):
+cdef parse_atom_ident_constants(pos, PyrexScanner s):
```

Comment:
I've renamed the and made them behave a little more like every other `p_...` function. I'd originally not used `p_...` because they seemed slightly different, but you're probably right that they can be made similar enough